test_coco.py

```diff
@@ -266,15 +270,59 @@ def test(data,
             cocoEval.accumulate()
             cocoEval.summarize()
             map, map50 = cocoEval.stats[:2]  # update results (mAP@0.5:0.95, mAP@0.5)
```

Comment:
Omg i realise `map` is a bad variable name to use because its a keyword in python. Can shun bian change it pwease? :D